onnx/optimizer.py

```diff
@@ -10,18 +10,36 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import onnx
 import onnx.onnx_cpp2py_export.optimizer as C  # type: ignore
+from onnx import ModelProto
 
 """Apply the optimization on the serialized ModelProto.
 
 Arguments:
-    input (string): serialized ModelProto
+    input (string or ModelProto): (serialized) model
     names (list of string): list of optimization names
 
+Return:
+    return (string or ModelProto depends on input) (serialized) optimized model
+
 Supported pass names:
     -- nop
     -- eliminate_nop_transpose
     -- fuse_consecutive_transposes
     -- fuse_transpose_into_gemm
 """
-optimize = C.optimize
+def optimize(model, pass_list=None):
+    if pass_list is None or len(pass_list) == 0:
+        pass_list = ['eliminate_nop_transpose',
```

Comment:
put this list of pass names in a tuple and set it as the default value of pass_list function argument?